src/layout/layout.js

```diff
@@ -370,11 +466,16 @@
       if (this.drawer_) {
         var drawerButton = this.element_.querySelector('.' +
           this.CssClasses_.DRAWER_BTN);
+        var drawerNav = this.drawer_.querySelector('.mdl-navigation');
+        if (drawerNav) {
+          drawerNav.setAttribute('role', 'menu');
```

Comment:
Its needed for [accessibility](https://www.w3.org/TR/wai-aria-practices-1.1/#menu). I didn't want to force all users of MDL lite to set this manually. Some aria related attributes are already set via js as well. I thought of it from the perspective of providing a menu layout that handles common aspects.